src/readfish/_utils.py

```diff
@@ -305,24 +310,36 @@ def draw_flowcell_split(
     00XX
     00XX
 
+
     :param flowcell_size: Number of channels on the flow cell
     :param split: The number of regions to split into, defaults to 1
     :param index: The index of the region to highlight, defaults to 0
+    :param prefix: Any leading string character to put on the row. Defaults to \t
     :return: String representation of the flowcell in ASCII art
     """
-    depth, width = get_flowcell_array(flowcell_size).shape
-    depth = round((depth / 2) + 0.5)
+    height, width = get_flowcell_array(flowcell_size).shape
+    height = round((height / 2) + 0.5)
+    # Flongle is a truly dumb shape - 10 rows of 13 columns, breaks maths above so just special case it
+    if flowcell_size == 126:
+        height -= 1
     cells = []
-    for _h in range(depth):
+    for _h in range(height):
         row = [
-            "    ",
+            prefix,
         ]
         for _w in range(width):
             row.append(".")
         cells.append(row)
     cells = np.array(cells)
     region = generate_flowcell(flowcell_size, split, axis)[index]
+    col, row = None, None
     for pos in region:
+        # Flongle has four channels which are "0", but are just placeholders
+        # in the array where there is no channel due to it's weird shape.
```

Comment:
This is an abomination of all that is good in the world. How can 0 possibly be used in this way? It's a travesty.

Concerns I have are that if channels in the future on other devices begin with a 0th channel will we have an issue? Should we do a double check where we confirm the flowcell type from the device?

Reply:
We query the number of channels over the API and infer from that atm